core/Pos/Binary/Class/Primitive.hs

```diff
@@ -3,110 +3,135 @@
 -- | Useful functions for serialization/deserialization.
 
 module Pos.Binary.Class.Primitive
-       (
-       -- * Primitives for serialization
-         putWord8S
-       , putBytesS
-
-       -- * Bi to SafeCopy
-       , getCopyBi
-       , putCopyBi
-       -- * Binary serialization
-       , AsBinary (..)
-       , AsBinaryClass (..)
-       , fromBinaryM
-
-       -- * Serialization with length
-       , putWithLength
-       , putWithLengthS
-       , getWithLength
-       , getWithLengthLimited
-       , putSmallWithLength
-       , putSmallWithLengthS
-       , getSmallWithLength
-
-       -- * Serialization strings and sequence of bytes
-       , getBytes
-       , putBytes
-
-       , getAsciiString1b
-       , putAsciiString1b
-       , sizeAsciiString1b
-
-       , Raw (..)
-       ) where
-
+    ( serialize
```

Comment:
Oh man! Enforcing this all by hand is a bit of a pain! Do we have some Elisp module for Emacs to format the project on save according to the Cardano Style guides? I would kill for such module 😉